src/System.Drawing.Primitives/src/System/Drawing/KnownColor.cs

```diff
@@ -45,9 +46,11 @@ public enum KnownColor
         Window,
         WindowFrame,
         WindowText,
+        */
 
         // "Web" Colors
-        Transparent,
+        FirstColor,
+        Transparent = FirstColor,
```

Comment:
These values should no longer escape, they are purely an implementation convenience.